### DISCUSSION TITLE
Wsl compatibility

### DISCUSSION THREAD

**Discussion by hudsonsferreira** [cline/cline#443] [Feature Requests]
It would be amazing if claude dev has wsl compatibility

**Comment by TomLucidor** [comment]
Seconding this since some packages refuses to comply with PowerShell

**Comment by arturskruze** [comment]
Cline won't be able to view the command's output. Please update VSCode (CMD/CTRL + Shift + P → "Update") and make sure you're using a supported shell: zsh, bash, fish, or PowerShell (CMD/CTRL + Shift + P → "Terminal: Select Default Profile"). [Still having trouble?](https://github.com/cline/cline/wiki/Troubleshooting-%E2%80%90-Shell-Integration-Unavailable)

This is for WSL Terminal. Definitely would be great if Cline could check WSL terminal.

**Comment by HittmanA** [comment]
Any update on this from Cline? Would be extremely useful.

**Comment by shimongewirtz** [comment]
still having issues with wsl

**Comment by yalexx** [comment]
When i try to set default terminal to WSL I'm getting 
Shell Integration Unavailable

Cline may have trouble viewing the command's output. Please update VSCode (`CMD/CTRL + Shift + P` → "Update") and make sure you're using a supported shell: zsh, bash, fish, or PowerShell (`CMD/CTRL + Shift + P` → "Terminal: Select Default Profile"). [Still having trouble?](https://github.com/cline/cline/wiki/Troubleshooting-%E2%80%90-Shell-Integration-Unavailable)
